api/src/main/java/net/kyori/adventure/text/Component.java

```diff
@@ -131,14 +131,27 @@
    * ---------------------------
    */
 
+  /**
+   * Creates a block NBT component builder.
+   *
+   * @return a builder
+   * @since 4.4.0
+   */
+  static BlockNBTComponent.@NonNull Builder blockNBTBuilder() {
+    return new BlockNBTComponentImpl.BuilderImpl();
+  }
+
+
   /**
    * Creates a block NBT component builder.
    *
    * @return a builder
    * @since 4.0.0
+   * @deprecated for removal since 4.4.0, see {@link #blockNBTBuilder()}
```

Comment:
Or this?
```suggestion
   * @deprecated for removal since 4.4.0, in favour of {@link #blockNBTBuilder()}
```

Reply:
i was just trying to match existing deprecations :p